Godeps/Godeps.json

```diff
@@ -77,7 +77,7 @@
 		},
 		{
 			"ImportPath": "github.com/mattn/go-isatty",
-			"Rev": "66b8e73f3f5cda9f96b69efd03dd3d7fc4a5cdb8"
+			"Rev": "a5cdd64afdee435007ee3e9f6ed4684af949d568"
```

Comment:
are there any other deps we should be updating at this time to match build.gradle versions?

Reply:
no this is the only one i found out to change.